internal/engine/actions/remediate/pull_request/pull_request.go

```diff
@@ -77,6 +80,49 @@ type Remediator struct {
 	bodyTemplate  *htmltemplate.Template
 }
 
+// prSlug is a string that identifies a pull request
+type prSlug string
+
+func (s prSlug) GetOrganization() string {
```

Comment:
Does it make sense to have a compiled regex that extracts named fields for these three parts?  RE2 is pretty efficient, and might be able to avoid scanning the strings multiple times and making extra copies of the fields compared with multiple splits and trims.

Reply:
OK, I've added the regex and sim;ified the slug to a single Parse() method